lms/djangoapps/teams/views.py

```diff
@@ -33,7 +34,16 @@ def get(self, request, course_id):
                 not has_access(request.user, 'staff', course, course.id):
             raise Http404
 
-        context = {"course": course}
+        context = {
+            "course": course,
+            "header_info": {
+                "description": _(
+                    "Course teams are organized into topics created by course instructors. Try"
```

Comment:
I'm thinking that this should be client-side. The ``TeamsTabView`` can construct a header for itself and pass these parameters to it.